tools/docker/Dockerfile.ubuntu18.04

```diff
@@ -75,7 +75,7 @@ ENV OCCLUM_RUST_VERSION=nightly-2020-09-08
 RUN curl https://sh.rustup.rs -sSf | \
         sh -s -- --default-toolchain ${OCCLUM_RUST_VERSION} -y && \
     rm -rf /root/.cargo/registry && rm -rf /root/.cargo/git && \
-    cargo -V
+    cargo install sccache
```

Comment:
I think it is good to have `cargo -V` to see the version installed.  How about keeping `cargo -V`?